src/main/java/org/jabref/logic/util/FileType.java

```diff
@@ -26,6 +26,7 @@
     DIN_1505(Localization.lang("%0 file", "DIN 1505"), "rtf"),
     ENDNOTE(Localization.lang("%0 file", "EndNote/Refer"), "ref", "enw"),
     ENDNOTE_XML(Localization.lang("%0 file", "EndNote XML"), "xml"),
+    ENDNOTE_XMP(Localization.lang("%0 file", "EndNote Xmp"), "xmp"),
```

Comment:
Why "Endnote XMP"? What has "Endnote" todo with XMP? I would just use "XMP", because it is an [ISO Standard](https://en.wikipedia.org/wiki/Extensible_Metadata_Platform), not something from Endnote.